frontend/dockerfile/docs/syntax.md

```diff
@@ -25,6 +25,52 @@ incrementing the major component of a version and you may want to pin the image
 change in between releases on labs channel, the old versions are guaranteed to be backward compatible.
 
 
+## Linked copies `COPY --link`, `ADD --link`
+
+To use this flag set Dockerfile version to at least `1.4`.
+
+```
```

Comment:
can you add `dockerfile` code-hints to the blocks that show `Dockerfile` examples? (so that highlighting will work if we publish it)